src/Styles/Page.css

```diff
@@ -0,0 +1,128 @@
+:root {
+  --card-color: #8ca8cc;
+}
+#page {
+  position: absolute;
+  margin: 5%;
+  padding: 2em;
+  background-color: rgba(255, 255, 255, 0.644);
```

Comment:
You're using some very specific transparency values in your color choices here, perhaps create a few more CSS variables and use them instead?